core/src/main/java/org/elasticsearch/common/xcontent/ObjectParser.java

```diff
@@ -64,6 +64,7 @@
         };
     }
 
+    private final Map<String, FieldParser> fieldParserMap = new HashMap<>();
```

Comment:
Moved this up from the middle of the class so it could be next to the other fields.